packages/cdk8s-cli/package.json

```diff
@@ -34,33 +34,31 @@
     "@types/fs-extra": "^8.1.1",
     "@types/jest": "^26.0.19",
     "@types/json-schema": "^7.0.6",
-    "@types/node": "^10.17.50",
-    "@typescript-eslint/eslint-plugin": "^4.11.1",
-    "@typescript-eslint/parser": "^4.11.1",
```

Comment:
How come these were downgraded?

Reply:
They are hard-coded in projen so whenever you run projen they get updated. But the important part is `yarn.lock`, no?